src/util/fees.cpp

```diff
@@ -26,15 +28,31 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
+const std::map<std::string, FeeEstimateMode>& FeeModeMap() {
```

Comment:
Is there any benefit to the function? (As opposed to a global map)

Reply:
The windows build (AppVeyor) fails to generate the map if I don't use a function. :/